pkg/kudoctl/cmd/install/install.go

```diff
@@ -181,59 +207,95 @@ func verifySingleFramework(name, previous string, gc *github.Client, k2c *kudo.C
+func installSingleFrameworkToCluster(name, path string, kc *kudo.Client) error {
+	frameworkPath := path + "/" + name + "-framework.yaml"
+	frameworkYamlFile, err := os.Open(frameworkPath)
+	if err != nil {
+		return errors.Wrap(err, "failed opening framework file")
+	}
+
+	frameworkByteValue, err := ioutil.ReadAll(frameworkYamlFile)
 	if err != nil {
-		return errors.Wrapf(err, "getting %s-framework.yaml", name)
+		return errors.Wrap(err, "failed reading framework file")
 	}
-	_, err = k2c.InstallFrameworkObjToCluster(frameworkYaml)
+
+	var f v1alpha1.Framework
+	err = yaml.Unmarshal(frameworkByteValue, &f)
+	if err != nil {
+		return errors.Wrapf(err, "unmarshalling %s-framework.yaml content", name)
+	}
+
+	_, err = kc.InstallFrameworkObjToCluster(&f)
 	if err != nil {
 		return errors.Wrapf(err, "installing %s-framework.yaml", name)
 	}
-	fmt.Printf("framework.%s/%s created\n", frameworkYaml.APIVersion, frameworkYaml.Name)
+	fmt.Printf("framework.%s/%s created\n", f.APIVersion, f.Name)
 	return nil
 }
 
 // Todo: needs testing
 // installSingleFrameworkVersionToCluster installs a given FrameworkVersion to the cluster
-func installSingleFrameworkVersionToCluster(name, path string, gc *github.Client, k2c *kudo.Client) error {
-	frameworkVersionYaml, err := gc.GetFrameworkVersionYaml(name, path)
+func installSingleFrameworkVersionToCluster(name, path string, kc *kudo.Client) error {
+	frameworkVersionPath := path + "/" + name + "-frameworkversion.yaml"
+	frameworkVersionYamlFile, err := os.Open(frameworkVersionPath)
 	if err != nil {
-		return errors.Wrapf(err, "getting %s-framework.yaml", name)
+		return errors.Wrap(err, "failed opening frameworkversion file")
 	}
-	_, err = k2c.InstallFrameworkVersionObjToCluster(frameworkVersionYaml)
+
+	frameworkVersionByteValue, err := ioutil.ReadAll(frameworkVersionYamlFile)
 	if err != nil {
-		return errors.Wrapf(err, "installing %s-framework.yaml", name)
+		return errors.Wrap(err, "failed reading frameworkversion file")
 	}
-	fmt.Printf("frameworkversion.%s/%s created\n", frameworkVersionYaml.APIVersion, frameworkVersionYaml.Name)
+
+	var fv v1alpha1.FrameworkVersion
+	err = yaml.Unmarshal(frameworkVersionByteValue, &fv)
+	if err != nil {
+		return errors.Wrapf(err, "unmarshalling %s-frameworkversion.yaml content", name)
+	}
+	_, err = kc.InstallFrameworkVersionObjToCluster(&fv)
+	if err != nil {
+		return errors.Wrapf(err, "installing %s-frameworkversion.yaml", name)
+	}
+	fmt.Printf("frameworkversion.%s/%s created\n", fv.APIVersion, fv.Name)
 	return nil
 }
 
 // Todo: needs testing
 // installSingleInstanceToCluster installs a given Instance to the cluster
-func installSingleInstanceToCluster(name, previous, path string, gc *github.Client, k2c *kudo.Client) error {
-	instanceYaml, err := gc.GetInstanceYaml(name, path)
+func installSingleInstanceToCluster(name, previous, path string, kc *kudo.Client) error {
+	frameworkInstancePath := path + "/" + name + "-instance.yaml"
```

Comment:
There's enough of these now to maybe even pull out as a utils package with a func for constructing these paths and names.